tests/drink/Cargo.toml

```diff
@@ -14,7 +14,7 @@ ink-wrapper-types = { path = "../../ink-wrapper-types", default-features = false
     "drink",
 ] }
 ink_primitives = "4.3.0"
-drink = "0.8.6"
```

Comment:
What are we loosing by going back?

Reply:
`0.8.5` bumped cc crates, which were then broken 
`0.8.6` gave `mut session: Session` argument to the `drink::test` functions

Reply:
Ah, the macro. That's terrible news b/c it means that all tests written for ink-wrapper-types 0.8.0 will require "downgrading" - ie re-introduction of `let mut session = ...`. 

Can we fix it?

Reply:
Resolved offline -- we will try to fix drink! first, so that it works with ink!4, release `0.8.7` and then return here.

Reply:
on it